ci/no-superuser/create-no-superuser-sqlserver.sh

```diff
@@ -21,7 +21,12 @@ do
 
     echo "INFO: Retry count: ${COUNT}"
 
-    docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P ${SQL_SERVER_PASSWORD} -d master -Q "SELECT 1"
+    if [[ "$1" = "sqlserver22" ]]; then
+        docker exec -t ${SQL_SERVER_CONTAINER_NAME} /opt/mssql-tools18/bin/sqlcmd -S localhost -U sa -P ${SQL_SERVER_PASSWORD} -d master -C -Q "SELECT 1"
```

Comment:
If the target SQL Server is `2022`, we have to use the new command `/opt/mssql-tools18/bin/sqlcmd`.